### PR TITLE
Correct localisation of energy consumption values

### PR DESCRIPTION
**Pull request by michaelsproul**
Fix the display of the 2601 MWh/year value for Ethereum's energy consumption.

## Description

Presently on the English page, 2601 MWh is written as 2.601 MWh, which looks 1000x too small because of the decimal point. It also doesn't match the given figure in TWh, or the number from the [report](https://carbon-ratings.com/eth-report-2022):

> According to our estimates, the Merge reduces the annualized electricity consumption of the Ethereum network from 22,900,320 MWh to 2,600.86 MWh by over 99.988 %.

It looks like this occurred due to some localisation confusion. In several European languages the roles of `.` and `,` are switched, meaning that `2.601 MWh` _is_ correct in e.g. German. I've attempted to correct the German and French translations to the best of my ability but would appreciate a check from someone who is fluent. The German translation quoted the TWh figure 10x too high and the French translation was using English conventions.

### CODE REVIEW

**Review by jmcook1186 — APPROVED** [review]
hi  thanks a lot for clearing this up!

File: src/content/translations/de/energy-consumption/index.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Bei Ethereum ist zum Beispiel der Transaktionsdurchsatz nicht nur der der Basise
...
+[CCRI](https://carbon-ratings.com) (Crypto Carbon Ratings Institute) hat den Stromverbrauch und den CO2-Fußabdruck des Ethereum-Netzwerks umfassend analysiert (siehe Bericht [_ Der Zusammenschluss - Auswirkungen auf den Stromverbrauch und den CO2-Footprint des Ethereum-Netzwerks_](https://carbon-ratings.com/eth-report-2022)). CCRI hat den Stromverbrauch verschiedener Nodes mit verschiedenen Hardware- und Client-Software-Konfigurationen gemessen. Dies führte zu einer Schätzung von **2.601 MWh** (0,0026 TWh) für den jährlichen Stromverbrauch des Netzes zum Zeitpunkt der Analyse (September 2022). Das entspricht den jährlichen CO2-Emissionen von **870 Tonnen CO2e** unter Berücksichtigung regionaler spezifischer Faktoren der CO2-Intensität.
```
Comment (jmcook1186) [inline src/content/translations/de/energy-consumption/index.md:21]:
```suggestion
[CCRI](https://carbon-ratings.com) (Crypto Carbon Ratings Institute) hat den Stromverbrauch und den CO2-Fußabdruck des Ethereum-Netzwerks umfassend analysiert (siehe Bericht [_ Der Zusammenschluss - Auswirkungen auf den Stromverbrauch und den CO2-Footprint des Ethereum-Netzwerks_](https://carbon-ratings.com/eth-report-2022)). CCRI hat den Stromverbrauch verschiedener Nodes mit verschiedenen Hardware- und Client-Software-Konfigurationen gemessen. Dies führte zu einer Schätzung von **2601 MWh** (0,0026 TWh) für den jährlichen Stromverbrauch des Netzes zum Zeitpunkt der Analyse (September 2022). Das entspricht den jährlichen CO2-Emissionen von **870 Tonnen CO2e** unter Berücksichtigung regionaler spezifischer Faktoren der CO2-Intensität.
```

**Comment by jmcook1186** [comment]
pinging @lukassim to see if we can get french/german speakers to check this

**Review by corwintines — APPROVED** [review]
Checked https://discord.com/channels/714888181740339261/885883808392572963/1072473937994338374 and https://discord.com/channels/714888181740339261/884063728742567978/1072473652085403678 to confirm German and French. 

Thanks @michaelsproul!

**Pull request merged**